pkg/cmd/repo/edit/edit.go

```diff
@@ -155,6 +206,269 @@ func editRun(ctx context.Context, opts *EditOptions) error {
 	return g.Wait()
 }
 
+func interactiveChoice(r *api.Repository) ([]string, error) {
+	options := []string{"Description", "Home Page URL", "Topics", "Default Branch Name", "Wikis", "Issues", "Projects", "Visibility", "Merge Options", "Template Repository"}
+
+	if r.IsInOrganization {
+		options = append(options, "Allow Forking")
+	}
+
+	qs := []*survey.Question{
+		{
+			Name: "Choice",
+			Prompt: &survey.MultiSelect{
+				Message: "What do you want to edit?",
+				Options: options,
+			},
+		},
+	}
+
+	var answers []string
+
+	err := prompt.SurveyAsk(qs, &answers, survey.WithPageSize(11))
+	if err != nil {
+		return []string{}, err
+	}
+
+	return answers, nil
+}
+
+func interactiveRepoEdit(r *api.Repository, topics []string) (repoInput *EditRepositoryInput, addTopics, removeTopics []string, err error) {
+	var defaultTopics string
+	for k, v := range topics {
+		if k == len(topics)-1 {
+			defaultTopics += v
+		} else {
+			defaultTopics += v + ","
+		}
+	}
+
+	choices, err := interactiveChoice(r)
+	if err != nil {
+		return nil, nil, nil, err
+	}
+
+	var qs []*survey.Question
+	var defaultMergeOptions []int
+	var defaultMergeSurvey []string
+
+	if r.MergeCommitAllowed {
+		defaultMergeOptions = append(defaultMergeOptions, 0)
+		defaultMergeSurvey = append(defaultMergeSurvey, "Allow Merge Commits")
+	}
+
+	if r.SquashMergeAllowed {
+		defaultMergeOptions = append(defaultMergeOptions, 1)
+		defaultMergeSurvey = append(defaultMergeSurvey, "Allow Squash Merging")
+	}
+
+	if r.RebaseMergeAllowed {
+		defaultMergeOptions = append(defaultMergeOptions, 2)
+		defaultMergeSurvey = append(defaultMergeSurvey, "Allow Rebase Merging")
+	}
+
+	if len(choices) > 0 {
+		for _, c := range choices {
+			switch c {
+			case "Description":
+				qs = append(qs, &survey.Question{
+					Name: "repoDescription",
+					Prompt: &survey.Input{
+						Message: "Description of the repository",
+						Default: r.Description,
+					},
+				})
+			case "Home Page URL":
+				qs = append(qs, &survey.Question{
+					Name: "repoURL",
+					Prompt: &survey.Input{
+						Message: "Repository home page URL?",
+						Default: r.HomepageURL,
+					},
+				})
+			case "Topics":
+				qs = append(qs, &survey.Question{
+					Name: "addTopics",
+					Prompt: &survey.Input{
+						Message: "Add topics?(csv format)",
+						Default: defaultTopics,
```

Comment:
Here we show the user his existing topics so that he needn't add them again, and thereby can delete what he doesn't need. Just like how the platform does on the UI.